lib/src/api/engine/remote/http/mod.rs

```diff
@@ -342,7 +344,8 @@ async fn router(
 			let ns = match ns {
 				Some(ns) => match HeaderValue::try_from(&ns) {
 					Ok(ns) => {
-						request = request.header("NS", &ns);
+						// This is a correct header value so the unwrap should be fine.
+						request = request.header("NS", &ns).unwrap();
```

Comment:
I unwrapped here because "NS" is a correct header name and the header value is already checked to be correct, so at this point  the `header` method should never return an error.

Reply:
Yeah I suppose it works. But as a rule of thumb I prefer having errors handled. So when we get a message like
```
Database failed: Something failed: Error: Error in header: hopefully a useful message that the request did not contain a namespace in the header
```
Then we can trace to a single line (or perhaps a small number of lines) where that error message exists. Otherwise with an unwrap, we cannot do that easily without a stack trace. And if there is not header - it is likely that someone is doing something unexpected so it won't occur to us. Or perhaps it was forgotten to be added in tests, but we will just get a panic.